server/license_test.go

```diff
@@ -1,36 +0,0 @@
-package main
```

Comment:
Removed this file because this is already tested in the mattermost-plugin-api, so it would be just duplicated code.